docs/sources/api/_index.md

```diff
@@ -34,19 +34,17 @@ These endpoints are exposed by the querier and the query frontend:
 - [`GET /loki/api/v1/series`](#list-series)
 - [`GET /loki/api/v1/index/stats`](#index-stats)
 - [`GET /loki/api/v1/tail`](#stream-log-messages)
-- [`POST /loki/api/v1/push`](#push-log-entries-to-loki)
-- [`GET /ready`](#identify-ready-loki-instance)
-- [`GET /metrics`](#return-exposed-prometheus-metrics)
```

Comment:
If `GET/ready` and `GET/metrics` are exposed by all components, shouldn't you add them to the list of endpoints exposed by all components that starts at line 19?

Reply:
They are already listed at line 21 and 22.

Reply:
Apologies. That's what I get for focusing on the change that GitHub highlights.  =P